src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -411,62 +411,35 @@ internal virtual NamedTypeSymbol GetNativeIntegerType(NamedTypeSymbol underlying
             throw ExceptionUtilities.Unreachable();
         }
 
-        /// <summary>
-        /// Figure out if the target runtime supports default interface implementation.
-        /// </summary>
+        bool IAssemblySymbolInternal.IsStaticClass(INamedTypeSymbolInternal namedType)
+            => namedType is { TypeKind: TypeKind.Class, IsStatic: true };
```

Comment:
the logic here is pulled into C# and VB separately as the two languages were doing the check slightly differently in the previous code i was updating.  so this keeps things preserved in that regard.